functions/changeNicknameInDatabase.php

```diff
@@ -0,0 +1,17 @@
+<?php
+function changeNicknameInDatabase(int $id, ?string $nickname, PDO $db) {
```

Comment:
Doc block!

You can also return type hint this one as void

Reply:
Done